src/memory.h

```diff
@@ -11,7 +11,8 @@
 #include <SDL.h>
 
 uint8_t read6502(uint16_t address);
-uint8_t real_read6502(uint16_t address, bool debugOn, uint8_t bank);
+uint8_t debug_read6502(uint16_t address, int16_t bank);
+uint8_t real_read6502(uint16_t address, bool debugOn, bool useBank, uint8_t bank);
```

Comment:
This allows for 

```
real_read6502(address, debugOn, true, -1)
```

by accident, which will read from bank 255. Using one argument and making -1 the sentinel value for "use the current bank" seems safer.